Assets/com.gdcbd.bossbattle/Scripts/Components/ReloadCurrentScene.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace com.gdcbd.bossbattle.components
+{
+    public class ReloadScene : MonoBehaviour
```

Comment:
We should utilize our Scene Loading package instad of directly calling these. Also use addressable scenes.